emojis.py

```diff
@@ -0,0 +1,12 @@
+def jslen(string):
+    return int(len(string.encode(encoding='utf_16_le'))/2)
```

Comment:
My only concern is that I don't understand why we need to specify the encoding to be `utf_16_le` when we are using utf-8 for everything.

Reply:
Hi team, this feat does not work with utf 8